server/Pipfile

```diff
@@ -12,6 +12,11 @@ psycopg2-binary = "*"
 flask-testing = "*"
 flask-restx = "*"
 python-dotenv = "*"
+google-auth = "*"
+requests = "*"
+oauthlib = "==3.0.1"
```

Comment:
Is this needed?

Reply:
yes, except for oauthlib, will remove that